webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/HttpsDataVault.scala

```diff
@@ -0,0 +1,104 @@
+package com.scalableminds.webknossos.datastore.datavault
+
+import com.scalableminds.webknossos.datastore.storage.{
+  FileSystemCredential,
+  HttpBasicAuthCredential,
+  LegacyFileSystemCredential,
+  RemoteSourceDescriptor
+}
+
+import java.net.URI
+import scala.concurrent.duration.DurationInt
+import sttp.client3._
+import sttp.model.Uri
+
+class HttpsDataVault(credential: Option[FileSystemCredential]) extends DataVault {
+
+  private val connectionTimeout = 5 seconds
+  private val readTimeout = 1 minute
```

Comment:
let’s go higher, I’d say 1 minute and 10 minutes respectively. You wouldn’t believe how slowly some servers answer :D

Reply:
These were the values from the original httpsfilesystem (https://github.com/scalableminds/webknossos/blob/6b5143a94e657c624370d0de85e36d6e64eeb97d/webknossos-datastore/app/com/scalableminds/webknossos/datastore/storage/httpsfilesystem/HttpsSeekableByteChannel.scala) but sure can be changed